### PR TITLE
Sample project to generate documents with configured privileges

### PR DESCRIPTION
**Pull request by avin101**
Sample application to generate a pdf document using the privileges which are configured using File Security api's.

### CODE REVIEW

File: sample/Kmd.Logic.FileSecurity.Client.DocumentPrivilegesSample/Program.cs [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+            Console.WriteLine(
+                "Configuration recieved successfully. SignConfiguration ID: {0} \nSignConfiguration Name: {1}\nSubscription ID : {2}",
+                signConfigurationResult.Id,
+                signConfigurationResult.Name,
+                signConfigurationResult.SubscriptionId);
+
+            // Generate document
+            Log.Information("Generate document using privileges...");
+            string pdfSample = configuration.SignConfigurationDetails.PdfEmptySampleLocation;
+            using (Document document = new Document(pdfSample))
```
Comment (smahapatra86) [inline sample/Kmd.Logic.FileSecurity.Client.DocumentPrivilegesSample/Program.cs]: Can we use the new `using` syntax here? e.g. `using (Document document = new Document(pdfSample));`

File: sample/Kmd.Logic.FileSecurity.Client.DocumentPrivilegesSample/Program.cs [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+                document.Save(configuration.SignConfigurationDetails.PdfGeneratedDocumentLocation + "pdf_with_privileges.pdf");
+                Log.Information(
+                    "Document with configured privileges generated successfully at {location}",
+                    configuration.SignConfigurationDetails.PdfGeneratedDocumentLocation + "pdf-with-privileges.pdf");
+            }
+        }
+
+        private static DocumentPrivilege FillPrivileges(PdfPrivilegeModel pdfPrivilege)
+        {
+            DocumentPrivilege privileges = DocumentPrivilege.ForbidAll;
```
Comment (smahapatra86) [inline sample/Kmd.Logic.FileSecurity.Client.DocumentPrivilegesSample/Program.cs]: Why we are doing this?

**Review by smahapatra86 — CHANGES_REQUESTED** [review]

File: sample/Kmd.Logic.FileSecurity.Client.DocumentPrivilegesSample/Program.cs (hunk 1 above)
Comment (avin101) [reply sample/Kmd.Logic.FileSecurity.Client.DocumentPrivilegesSample/Program.cs]: Done

**Review by smahapatra86 — APPROVED** [review]

**Review by thecorrado — COMMENTED** [review]
I don't have any comments, just one suggestion: could you add one more example? Generating .docx document to the pdf document. Such example cover real-world scenario that we have with the docgen module :)

**Review by thecorrado — APPROVED** [review]

**Pull request merged**